boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig

```diff
@@ -0,0 +1,35 @@
+#CONFIG_RISCV=y
```

Comment:
Why is this commented out? (It might be correct, just trying to understand)

Reply:
It's seems can't be defined by user?
If opening the define, will cause this error:
error: RISCV (defined at arch/Kconfig:53) is assigned in a configuration file, but is not directly
user-configurable (has no prompt). It gets its value indirectly from other symbols.

So we select this symbol by selecting the ARCH_IS_SET symbol.